src/managers/GuildEmojiRoleManager.js

```diff
@@ -51,7 +51,7 @@ class GuildEmojiRoleManager extends DataManager {
       resolvedRoles.push(resolvedRole);
     }
 
-    const newRoles = [...new Set(resolvedRoles.concat(...this.cache.values()))];
+    const newRoles = [...new Set(resolvedRoles.concat(...this.cache.map(role => role.id)))];
```

Comment:
```suggestion
    const newRoles = [...new Set(resolvedRoles.concat(...this.cache.keys()))];
```
I believe this should work too